src/components/ui/LoginForm.tsx

```diff
@@ -72,18 +51,24 @@ export const LoginForm = () => {
 
   async function currentSession() {
     try {
-      const { accessToken, idToken } = (await fetchAuthSession()).tokens ?? {};
-      console.log('accessToken, idToken', accessToken, idToken);
+      const { idToken } = (await fetchAuthSession()).tokens ?? {};
+      console.log('idToken', idToken?.toString());
```

Comment:
This is how we get idToken for API gateway authentication. I will delete it after complete common API client function.